src/pages/react/Installations/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: React App Dependencies Installation 
+---
+## Pre-requisites 
+We need the right combinations of `Babel` ,` Webpack` and `NPM `dependencies to get started with a basic React Application.  `create-react-app` a command line tool is provided by React developers to abstract all the complexities involved in setting up a react project.
+What goes under the hood when we use this command is what we will be explaining here.
+
+```C
```

Comment:
This does not need to be in a code block. Simply using `1.` `2.` `3.` will render out the correct formatting for a list, like so:

1. one thing
2. another thing
3. a really awesome thing

You can see how formatting is added automatically through markdown